skt/executable.py

```diff
@@ -460,6 +461,50 @@ def cmd_report(cfg):
     reporter.report()
 
 
+def cmd_console_check(cfg):
+    """
+    Check the console logs for any traces.
+
+    Args:
+        cfg: A dictionary of skt configuration.
+    """
+    remove_oldresult(cfg.get('output_dir'), 'console_check.')
+    console_result_path = join_with_slash(cfg.get('output_dir'),
+                                          'console_check.result')
+    console_report_path = join_with_slash(cfg.get('output_dir'),
+                                          'console_check.report')
+
+    if not cfg.get('krelease') or not cfg.get('console'):
+        raise Exception('<krelease> or <console-url> parameter missing!')
+
+    console_logs = []
+    trace_list = []
+
+    for console_path_or_url in cfg.get('console'):
+        console_log = skt.console.ConsoleLog(cfg.get('krelease'),
+                                             console_path_or_url)
+
+        console_logs.append(console_log)
+        trace_list.append(console_log.gettraces())
+
+    if any(trace_list):
+        report_string = ''
+
+        for traces in trace_list:
+            if traces:
+                report_string += '{}\n{}:\n\n{}\n\n'.format(
+                    'This is the first trace we found in ',
+                    cfg.get('console')[trace_list.index(traces)],
+                    traces[0]
```

Comment:
Sorry, I'm lost, what are we outputting here?

Reply:
The path/URL of the console log in which the trace was found and the related first trace

Reply:
I see, thank you. Do you think you can explain how you get the path/URL in a comment above this line:

    cfg.get('console')[trace_list.index(traces)],